src/index.ts

```diff
@@ -371,9 +371,9 @@ export class LinkChecker extends EventEmitter {
     // The `retry-after` header can come in either <seconds> or
     // A specific date to go check.
     let retryAfter = Number(retryAfterRaw) * 1000 + Date.now();
-    if (isNaN(retryAfter)) {
+    if (Number.isNaN(retryAfter)) {
```

Comment:
Oh god these are semantically different.  Why JavaScript, why.  This is a good change, probably.

Reply:
This isn't enforced by the current rules AFAICT. I'll check if gts supports this tomorrow.

Reply:
Might be worth looking into https://github.com/sindresorhus/eslint-plugin-unicorn and enable any rules selectively or use their recommended config.